cmake/adjust/template/vendor/CXX_Clang.cmake

```diff
@@ -0,0 +1,51 @@
+include_guard(GLOBAL)
+
+# Debug
+set(NBL_CXX_DEBUG_COMPILE_OPTIONS
+	-ggdb3 -Wall -fno-omit-frame-pointer -fstack-protector-strong
+)
+
+# Release
+set(NBL_CXX_RELEASE_COMPILE_OPTIONS
+	-fexpensive-optimizations
+)
```

Comment:
I think O3 enables this by default ?

Reply:
yes however [`-fexpensive-optimizations`](<https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html#index-fexpensive-optimizations>) is not Clang's option but GCC's